lib/clients/rabbitmq.ex

```diff
@@ -103,4 +103,19 @@ defmodule ExRabbitPool.RabbitMQ do
         {:error, error}
     end
   end
+
+  @impl true
+  def qos(channel, queue, options \\ [])
+
+  def qos(_channel, _queue, []), do: :ok
+
+  def qos(channel, queue, prefetch_count: prefetch_count) do
+    Logger.warn(
+      "[ExRabbitPool.RabbitMQ.consume] queue: #{inspect(queue)} setting prefetch_count to #{
+        prefetch_count
+      }"
+    )
+
+    Basic.qos(channel, prefetch_count: prefetch_count)
```

Comment:
this should not be specific to `prefetch_count` we should pass through the options to `Basic.qos`